### PR TITLE
Adding enable-kv.sh script

### PR DESCRIPTION
**Pull request by venu-block1**
## Change Description
Adding enable-kv.sh script to eos/contracts

## Change Type
**Select ONE**
- [ ] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: contracts/enable-kv/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+1. nodeos -e -p eosio --plugin eosio::producer_plugin --plugin eosio::producer_api_plugin --plugin eosio::chain_api_plugin --plugin eosio::http_plugin --plugin eosio::history_plugin --plugin eosio::history_api_plugin --filter-on=* --access-control-allow-origin=* --contracts-console --http-validate-host=false --verbose-http-errors --max-transaction-time=1000 --backing-store chainbase --data-dir=datadir
+
+In the first terminal
+1. $EOS_3_0/contracts/enable-kv/enable-kv.sh -c $EOS_3_0/build/contracts/contracts/
+1. cd $EOSIO_CDT_3_0/examples/kv_map
+1. mkdir build
+1. cd build
+1. cmake .. -DCMAKE_PREFIX_PATH=$EOS_3_0/build -DEOSIO_CONTRACTS_ROOT=$EOSIO_CDT_3_0/build/contracts
+1. make
+1. cleos set contract eosio $EOSIO_CDT_3_0/examples/kv_map/build/kv_map -p eosio@active
```
Comment (nksanthosh) [inline contracts/enable-kv/README.md:19]: Better to use a non-privileged account here.

**Review by nksanthosh — CHANGES_REQUESTED** [review]

File: contracts/enable-kv/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Summary
+This README illustrastes the steps for usage of enable-kv.sh script with kv_map as an example. Set your environment variables as follows before you begin the rest of the steps.
```
Comment (nksanthosh) [inline contracts/enable-kv/README.md:2]: Some spelling issues? illustrastes

File: contracts/enable-kv/README.md (hunk 1 above)
Comment (venu-block1) [reply contracts/enable-kv/README.md]: Done.

File: contracts/enable-kv/README.md (hunk 2 above)
Comment (venu-block1) [reply contracts/enable-kv/README.md]: Fixed.

File: contracts/enable-kv/enable-kv.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    url=http://127.0.0.1:8888
+fi
+
+curl -X POST $url/v1/producer/schedule_protocol_feature_activations -d '{"protocol_features_to_activate": ["0ec7e080177b2c02b278d5088611686b49d739925a92d9bfcacd7fc6b74053bd"]}'
+sleep 1s
+cleos set contract eosio $contracts_dir/eosio.boot -p eosio@active
+sleep 1s
+#cleos set contract eosio $contracts_dir/eosio.bios -p eosio@active
+sleep 1s
+#cleos push action eosio setkvparams '[{"max_key_size":64, "max_value_size":1024, "max_iterators":128}]' -p eosio@active
```
Comment (linhuang-blockone) [inline contracts/enable-kv/enable-kv.sh:37]: Remove commented out code. I forgot to remove them before publishing it,

File: contracts/enable-kv/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Bring up nodeos in a different terminal
+1. export PATH=$EOS_3_0/build/bin:$PATH
+1. nodeos -e -p eosio --plugin eosio::producer_plugin --plugin eosio::producer_api_plugin --plugin eosio::chain_api_plugin --plugin eosio::http_plugin --plugin eosio::history_plugin --plugin eosio::history_api_plugin --filter-on=* --access-control-allow-origin=* --contracts-console --http-validate-host=false --verbose-http-errors --max-transaction-time=1000 --backing-store chainbase --data-dir=datadir
+
+In the first terminal
+1. $EOS_3_0/contracts/enable-kv/enable-kv.sh -c $EOS_3_0/build/contracts/contracts/
+1. cd $EOSIO_CDT_3_0/examples/kv_map
+1. mkdir build
+1. cd build
+1. cmake .. -DCMAKE_PREFIX_PATH=$EOS_3_0/build -DEOSIO_CONTRACTS_ROOT=$EOSIO_CDT_3_0/build/contracts
```
Comment (linhuang-blockone) [inline contracts/enable-kv/README.md:17]: $EOSIO_CDT_3_0/build does not have contracts directory. Set another environment for eos-contracts. 

Have you run through the steps?

File: contracts/enable-kv/README.md (hunk 1 above)
Comment (linhuang-blockone) [inline contracts/enable-kv/README.md]: Better use a non-eosio account like alice.

**Review by linhuang-blockone — CHANGES_REQUESTED** [review]

File: contracts/enable-kv/README.md (hunk 4 above)
Comment (venu-block1) [reply contracts/enable-kv/README.md:17]: The assumption is that eos is built in $EOSIO_CDT_3_0/build/ (which is the standard practice). We can follow the standard practice, or build $EOSIO_CDT_3_0/contracts/contracts separately in a different directory, and create another environment variable. 

What's the preference?

File: contracts/enable-kv/README.md (hunk 1 above)
Comment (venu-block1) [reply contracts/enable-kv/README.md]: Santhosh commented the same. And change for this already committed.

File: contracts/enable-kv/enable-kv.sh (hunk 3 above)
Comment (venu-block1) [reply contracts/enable-kv/enable-kv.sh]: Done.

File: contracts/enable-kv/README.md (hunk 4 above)
Comment (linhuang-blockone) [reply contracts/enable-kv/README.md:17]: In your '-DEOSIO_CONTRACTS_ROOT=$EOSIO_CDT_3_0/build/contracts' option, there is no such directory '$EOSIO_CDT_3_0/build/contracts'.

Comment (venu-block1) [reply contracts/enable-kv/README.md:17]: Apparently, cmake skipped processing -DEOSIO_CONTRACTS_ROOT (as the directory was not found), and rest of the steps went through fine. 

Not sure, if it is even required. Can you please confirm?

Comment (linhuang-blockone) [reply contracts/enable-kv/README.md:17]: Please take it and retest. If all steps pass, we are good.

File: contracts/enable-kv/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+## Summary
+This README illustrastes the steps for usage of enable-kv.sh script with kv_map as an example. Set your environment variables as follows before you begin the rest of the steps.
+1. export EOS_3_0=[eos 3.0 directory]
+1. export EOSIO_CDT_3_0=[eosio.cdt 3.0 directory]
```
Comment (nksanthosh) [inline contracts/enable-kv/README.md:4]: Please check the latest CDT version

File: contracts/enable-kv/README.md (hunk 4 above)
Comment (venu-block1) [reply contracts/enable-kv/README.md]: Removed unnecessary cmake command line option.

File: contracts/enable-kv/README.md (hunk 5 above)
Comment (venu-block1) [reply contracts/enable-kv/README.md]: Fixed it.

**Review by linhuang-blockone — APPROVED** [review]

**Review by nksanthosh — APPROVED** [review]

**Pull request merged**